### DISCUSSION TITLE
How to connect Stripe gateway

### DISCUSSION THREAD

**Discussion by gen035** [swellstores/community#293] [Help] [answered]
Hi! I am trying to integrate Stripe to my Swell account, I have a Stripe account ready to go as well as credentials, but cannot find how to connect them together. I just want to use the Swell hosted version. Thanks!

**Comment by logeshswell** [comment, marked as answer]
@gen035 There is a minor bug with the connection of Stripe payment gateway in Swell Admin dashboard. Can you follow this discussion for more details and updates on the fix?

**Reply by gen035** [reply]
Any ETA on a fix? It's basically the last step needed before I can go to prod with my website.

**Reply by logeshswell** [reply]
The fix has been made and it is yet to be deployed. I will let you know once it is deployed.
Regarding the ETA for the deployment, I can confirm with you shortly.

**Reply by gen035** [reply]
Thank you!
